infrastructure/images/k6-action/Dockerfile

```diff
@@ -57,16 +57,18 @@ WORKDIR /jsonnet/vendor
 RUN jb init && \
     jb install github.com/jsonnet-libs/k8s-libsonnet/${K8S_LIBSONNET_VERSION}@main
```

Comment:
Are k8s-libsonnet not needed in the final image?

Reply:
I'm guessing it gets installed in /usr/local/bin 🙈

Reply:
/jsonnet/vendor, but good catch!
https://github.com/Altinn/altinn-platform/blob/839e385ad3dc6749b0e8863a32599eb3636ac0c3/infrastructure/images/k6-action/Dockerfile#L52

Reply:
The reason why it's not failing it's because I'm not actually using it atm.
I ended up just relying on kubectl to generate the ConfigMap instead.